proto/storage/collector_runtime_config.proto

```diff
@@ -0,0 +1,62 @@
+syntax = "proto3";
+
+package storage;
+
+import "storage/resource_collection.proto";
+
+option go_package = "storage";
+option java_package = "io.stackrox.proto.storage";
+
+message CollectorRuntimeConfigACK {
+  string instance_id = 1;
+  Action action = 2;
+
+  enum Action {
+    ACK = 0;
+    NACK = 1;
+  }
+}
+
+enum RuntimeFeature {
+  PROCESSES = 0;
+  NETWORK_CONNECTIONS = 1;
+  LISTENING_ENDPOINTS = 2;
+}
+
+message Status {
+  message ProcessStatus {
+    bool enabled = 1;
+  }
+  
+  message NetworkConnectionStatus {
+    bool enabled = 1;
+    bool aggregate_external = 2;
+  }
+
+  message ListeningEndpointStatus {
+    bool enabled = 2;
+  }
+
+  oneof status {
+    ProcessStatus process_status = 1;
+    NetworkConnectionStatus network_connection_status = 2;
+    ListeningEndpointStatus listening_endpoint_status = 3;
+  }
+}
+
+message RuntimeFeatureConfig {
+
+  message RuntimeRule {
+    string resource_collection_name = 1;
+    Status status = 2;
+  }
+
+  RuntimeFeature feature = 1;
+  string default_status = 2;
```

Comment:
Since we now have structured statuses in rules, we probably want the default to be of the same type.

Reply:
Done

Reply:
Resolving as the comment was addressed and @ovalenti is on PTO.